querier.go

```diff
@@ -266,6 +267,62 @@ func (q *blockQuerier) Close() error {
 	return merr.Err()
 }
 
+// Bitmap used by func special to check whether a character needs to be escaped.
+var specialBytes [16]byte
+
+// special reports whether byte b needs to be escaped.
+func special(b byte) bool {
```

Comment:
In the context of the whole file and package, a more specific function name than `special()` would be good. How about `isRegexMetaCharacter()` and `specialBytes` -> `regexMetaCharacterBytes`?